python_modules/dagster/dagster/_core/telemetry.py

```diff
@@ -134,15 +135,18 @@ def _telemetry_wrapper(
             f"Attempted to log telemetry for function {f.__name__} that is not in telemetry whitelisted "
             f"functions list: {TELEMETRY_WHITELISTED_FUNCTIONS}."
         )
-
-    var_names = f.__code__.co_varnames
-    try:
-        instance_index = var_names.index("instance")
-    except ValueError as e:
+    sig = inspect.signature(f)
```

Comment:
what's going on here

Reply:
just translated the use of `__code__` to using `inspect` since the decorator util `update_wrapper` copies forward the properties that power inspect but not `__code__`.